challenge_6.js

```diff
@@ -6,10 +6,26 @@
  */
 exports.SumOfASquare = SumOfASquare;
 
-function SumOfASquare(naturalNumbers){
+function SumOfASquare(naturalNumbers) {
   this.naturalNumbers = naturalNumbers;
 }
 
+SumOfASquare.prototype.sumOfSquares = function() {
+  let sumSquared = 0;
+
+  for (let i = 0; i <= this.naturalNumbers; i++) {
+    let square = Math.pow(i, 2);
```

Comment:
You can skip the step of saving Math.pow(i,2) into `squared` variable and set it to the `sum` variable below.